qiskit/pulse/__init__.py

```diff
@@ -13,11 +13,202 @@
 # that they have been altered from the originals.
 
 """
+
 .. currentmodule:: qiskit.pulse
 
-===========================
-Pulse (:mod:`qiskit.pulse`)
-===========================
+
+Getting Started with OpenPulse
+==============================
+
+Qiskit Pulse programs give you more control than :py:class:`~qiskit.circuit.QuantumCircuit` s. At
+this level, you can specify the exact time dynamics of the input signals
+across all input channels. Most quantum algorithms can be described with
+circuits alone using gate operations – typically, it’s when you want to
```

Comment:
It might be nice instead to focus on the fundamental use of bootstrapping a quantum system into a quantum computer by implementing a universal basis gate set, and then tuning the resulting gates. Error mitigation and time dependence are definitely slightly different and worth mentioning but are less fundamental.

Reply:
yeah, I'd appreciate if we could update content in follow up works. I'd like to focus on migrating first

Reply:
My apologies, it wasn't clear to me this was just a migration.